frontend/assets/javascripts/src/modules/analytics/optimizely.js

```diff
@@ -20,8 +16,8 @@ define(function() {
     }
 
     function init() {
-        var userVariations = window.optimizely.data.state.variationMap || false;
-        if (userVariations && userVariations[SETTINGS.experimentId] === SETTINGS.variationId) {
+        var userVariations = window.optimizely.variationMap || false;
+        if (userVariations && userVariations[EXPERIMENT_ID] === 1) {
```

Comment:
Don't know much about the context (i.e. what a variation and a variation id is...), but I think it would read better if you were to replace the `1` with something like `VARIATION_ID`.